### DISCUSSION TITLE
Transcribe Lyrics

### DISCUSSION THREAD

**Discussion by Joelson-Forte** [Purfview/whisper-standalone-win#276] [Q&A] [answered]
Is there any possibility of transcribing audio files containing songs in the same way as Capcut Auto Lyrics feature?
https://www.youtube.com/watch?v=8ihzg8-y2Vg

I believe it is necessary to isolate the sounds of the music and vocal; For this, Demucs might be a good option. https://github.com/CarlGao4/Demucs-Gui

**Comment by Purfview** [comment, marked as answer]
It's already implemented, and better than Demucs -> https://github.com/Purfview/whisper-standalone-win/discussions/231

**Comment by ClaireCJS** [comment]
The problem is, the AI often guesses wrong words, even though a TXT file of lyrics already often exists.

Is there a way to prime it with the TXT file, so that it doesn't guess/hear the wrong words?

More specifically: If i have the TXT lyrics already, I want it to just add timing (and things like missing choruses, or lines missing from the lyrics).
